docs/graphx-programming-guide.md

```diff
@@ -235,8 +235,9 @@ triplet view of a graph to render a collection of strings describing relationshi
 val graph: Graph[(String, String), String] // Constructed from above
 // Use the triplets view to create an RDD of facts.
 val facts: RDD[String] =
-  graph.triplets.map(triplet =>
-    triplet.srcAttr._1 + " is the " + triplet.attr + " of " + triplet.dstAttr._1)
+  graph.triplets.map { triplet =>
```

Comment:
The brace change should be reverted here to match the other examples.
The rest seems OK as it's at least visible, and makes style more consistent.